build.sbt

```diff
@@ -162,7 +162,10 @@ lazy val userImpl = (project in file("user-impl"))
   .dependsOn(userApi)
   .settings(
     version := "1.0-SNAPSHOT",
-    libraryDependencies += lagomJavadslPersistenceCassandra
+    libraryDependencies ++= Seq(
+      lagomJavadslPersistenceCassandra,
+      lagomJavadslTestKit
+         )
```

Comment:
Watch out for indentation... not a big problem of course, but it helps readability to be consistent. IDEA can auto-indent for you: the default shortcut is control-alt-i (control-option-i on Mac).